app/views/upcoming_offline_reservation_mailer/generate_mail.html.haml

```diff
@@ -0,0 +1,10 @@
+%p
+  The instrument you have reserved for today at
+  #{l(@reservation.reserve_start_at, format: :timeonly)},
+  #{@reservation.product}, is down.
+
+  If the instrument is still down at your scheduled time, your reservation will
+  be canceled at no cost.
+
+%p
+  Please contact the #{t("facility_downcase")} for more information.
```

Comment:
Can we include a link to the facility? And possibly a link to the order detail/reservation?